applications/nrf_desktop/src/events/config_event.h

```diff
@@ -35,7 +35,7 @@ extern "C" {
 	X(TIMEOUT)			\
 	X(REJECT)			\
 	X(WRITE_ERROR)			\
-	X(DISCONNECTED_ERROR)
```

Comment:
Do the same rename on Python side to keep config channel state names consistent.
You may also update the `WRITE_ERROR`, since it also could be caused by disconnection.

Reply:
can write error happen in normal situation?

Reply:
done

Reply:
I think it could occur if we have ongoing GATT operation and disconnection occurs.